packages/cms-data-sync/src/utils/setup.ts

```diff
@@ -6,6 +6,9 @@ import { createClient } from 'contentful-management';
 
 import { getAccessTokenFactory, SquidexGraphql } from '@asap-hub/squidex';
 
+export const isVerbose = () =>
+  process.env.VERBOSE_DATA_SYNC && process.env.VERBOSE_DATA_SYNC == 'true';
```

Comment:
it seems the test is failing due to that in CI 
```suggestion
  process.env.VERBOSE_DATA_SYNC && process.env.VERBOSE_DATA_SYNC === 'true';
```